services/contract_call_local.proto

```diff
@@ -93,9 +93,21 @@ message ContractFunctionResult {
     repeated ContractLoginfo logInfo = 6;
 
     /**
-     * the list of smart contracts that were created by the function call
+     * [DEPRECATED] the list of smart contracts that were created by the function call.
+     * 
+     * The created ids will now _also_ be externalized through internal transaction 
+     * records, where each record has its alias field populated with the new contract's 
+     * EVM address. (This is needed for contracts created with CREATE2, since 
+     * there is no longer a simple relationship between the new contract's 0.0.X id 
+     * and its Solidity address.)
      */
-    repeated ContractID createdContractIDs = 7;
+    repeated ContractID createdContractIDs = 7 [deprecated=true];
+
+    /**
+     * The new contract's EVM address. Only populated after release 0.23, where 
+     * each created contract will have its own record.
+     */
+     bytes evm_address = 8;
```

Comment:
Just to confirm, this value will be populated regardless of CREATE vs CREATE2 or root contract or internal, right?

Reply:
Correct.